packages/react-vapor/src/components/refresh/RefreshCallback.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {IReactVaporState} from '../../ReactVapor';
+import {callIfDefined} from '../../utils/FalsyValuesUtils';
+import {IDispatch} from '../../utils/ReduxUtils';
+import {RefreshCallBackActions} from './RefeshCallbackActions';
+import {RefreshStatus, RefreshStatusSelectors} from './RefreshCallbackReducer';
+
+export interface IRefreshCallbackOwnProps {
+    id: string;
+    time?: number;
+    incremental?: number;
+    intervalInMs?: number;
+    callback: (start: () => void) => void;
+    renderCount?: (count: number) => React.ReactNode;
+}
+
+export interface IRefreshCallbackState {
+    count: number;
+}
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IRefreshCallbackOwnProps) => ({
+    status: RefreshStatusSelectors.getRefreshStatus(state, {id: ownProps.id}),
+});
+
+const mapDispatchToProps = (dispatch: IDispatch, ownProps: IRefreshCallbackOwnProps) => ({
+    start: () => dispatch(RefreshCallBackActions.start(ownProps.id)),
+    stop: () => dispatch(RefreshCallBackActions.stop(ownProps.id)),
+    inProgress: () => dispatch(RefreshCallBackActions.inProgress(ownProps.id)),
+});
+
+export interface IRefreshCallbackProps
```

Comment:
```suggestion
export type IRefreshCallbackProps = IRefreshCallbackOwnProps & 
   ReturnType<typeof mapStateToProps> & 
   ReturnType<typeof mapDispatchToProps>;
```

You don't need `Partial` for your mapStateToProps nor mapDispatchToProps